gsy_framework/data_classes.py

```diff
@@ -350,60 +350,60 @@ def csv_fields(cls) -> Tuple:
         """Return json string of the representation."""
         # __dict__ instead of asdict to not recursively deserialize objects
         trade_dict = deepcopy(self.__dict__)
-        trade_dict["offer_bid"] = trade_dict["offer_bid"].to_json_string()
+        trade_dict["order"] = trade_dict["order"].to_json_string()
         if key_in_dict_and_not_none(trade_dict, "residual"):
             trade_dict["residual"] = trade_dict["residual"].to_json_string()
-        if key_in_dict_and_not_none(trade_dict, "offer_bid_trade_info"):
-            trade_dict["offer_bid_trade_info"] = (
-                trade_dict["offer_bid_trade_info"].to_json_string())
+        if key_in_dict_and_not_none(trade_dict, "trade_orders_info"):
+            trade_dict["trade_orders_info"] = (
+                trade_dict["trade_orders_info"].to_json_string())
         return json.dumps(trade_dict, default=json_datetime_serializer)
 
     @classmethod
     def from_json(cls, trade_string) -> "Trade":
         """De-serialize trade from json string."""
         trade_dict = json.loads(trade_string)
-        trade_dict["offer_bid"] = BaseBidOffer.from_json(trade_dict["offer_bid"])
+        trade_dict["order"] = BaseOrder.from_json(trade_dict["order"])
         if trade_dict.get("residual"):
-            trade_dict["residual"] = BaseBidOffer.from_json(trade_dict["residual"])
+            trade_dict["residual"] = BaseOrder.from_json(trade_dict["residual"])
         if trade_dict.get("creation_time"):
             trade_dict["creation_time"] = str_to_pendulum_datetime(trade_dict["creation_time"])
         else:
             trade_dict["creation_time"] = None
         if trade_dict.get("time_slot"):
             trade_dict["time_slot"] = str_to_pendulum_datetime(trade_dict["time_slot"])
-        if trade_dict.get("offer_bid_trade_info"):
-            trade_dict["offer_bid_trade_info"] = (
-                TradeBidOfferInfo.from_json(trade_dict["offer_bid_trade_info"]))
+        if trade_dict.get("trade_orders_info"):
+            trade_dict["trade_orders_info"] = (
+                TradeOrdersInfo.from_json(trade_dict["trade_orders_info"]))
         return Trade(**trade_dict)
 
     @property
     def is_bid_trade(self) -> bool:
         """Check if the instance is a bid trade."""
-        return isinstance(self.offer_bid, Bid)
+        return isinstance(self.order, Bid)
 
     @property
     def is_offer_trade(self) -> bool:
         """Check if the instance is an offer trade."""
-        return isinstance(self.offer_bid, Offer)
+        return isinstance(self.order, Offer)
 
     def serializable_dict(self) -> Dict:
         """Return a json serializable representation of the class."""
         return {
             "type": "Trade",
-            "match_type": type(self.offer_bid).__name__,
+            "match_type": type(self.order).__name__,
             "id": self.id,
-            "offer_bid_id": self.offer_bid.id,
+            "order_id": self.order.id,
```

Comment:
I need to start a discussion about this change here. 
Although I am pretty sure that we do not use the id of the order in the d3a-web, neither for exposing to the UI nor for the results export yet, changing the `serializable_dict` will always have an impact to our DB; because we will have multiple versions of the serialised objects in the DB. Last time it was problematic, when the `time` member was renamed to `time_slot` (at the same time the meaning was changed).  
AFAIS, we have the following solutions to deal with this in the future:
1. Deal with all the different version of the serialized objects in the code (in this case it would be `if "order_id" in serialized_trade or "offer_bid_id" in serialized_trade: ...`
2. Add migrations and update all of the results in the DB
3. Do not change key names in the `serializable_dict` any more, only add new ones or delete these.

Solution 2 is not debatable AFAIS, because this migration would take days to process and will introduce errors in the DB.
Solution 1 is ugly IMO and you can taste some of it in the `ExportSimulationResults._write_csv_file_from_list_of_dicts` on gsy-web.
I would vote for solution 3. But I would also like to hear everybody's opinion about it.
@spyrostz , @fievelk , @faizan2590 , @Hassan754